core/Filesystem.php

```diff
@@ -37,6 +37,12 @@ public static function deleteAllCacheOnUpdate($pluginName = false)
         foreach ($plugins as $plugin) {
             $plugin->reloadPluginInformation();
         }
+
+        /**
+         * Triggered after all non-memory caches are cleared (eg, via the cache:clear
+         * command).
```

Comment:
Actually the `PiwikCache::flushAll();` above also clears the `Lazy`, `Transient` and `Eager` cache.

Reply:
Yes, but the event isn’t fired every time the in memory caches are cleared, that’s just a side effect of this method. The purpose of the filesystem method is to clear non-memory caches (and if you’re going to do that, it doesn’t make sense to keep the in memory ones.

I think adding that detail would eventually result in confusion.

Reply:
That's true ;)